File: glDemo/src/main/java/com/kanawish/glepisodes/module/AppModule.java
package com.kanawish.glepisodes.module;

import toothpick.config.Module;

/**
 */
public class AppModule extends Module {

    public AppModule() {
        // TODO: Add more toothpick bindings here.
        bind(ActivityHierarchyServer.class).toInstance(ActivityHierarchyServer.NONE);
    }
}

File: arDemo/src/main/java/com/kanawish/ardemo/di/AppModule.kt
package com.kanawish.ardemo.di

import android.app.Application
import com.kanawish.di.ActivityInjectionLifecycle
import toothpick.config.Module

class AppModule(appContext: Application) : Module() {

    init {
        bind(Application.ActivityLifecycleCallbacks::class.java).to(ActivityInjectionLifecycle::class.java)
//        bind(ScriptManager::class.java).to(FileSystemManager::class.java)
    }
}